tools/block-generator/generator/generate.go

```diff
@@ -371,21 +341,25 @@ func (g *generator) WriteBlock(output io.Writer, round uint64) error {
 			intra += numTxns
 		}
 		generated = time.Now()
+		g.reportData.Counters[BlockgenGenerateTimeMS] += uint64(generated.Sub(start).Milliseconds())
 
-		vBlock, ledgerTxnCount, err := g.evaluateBlock(cert.Block.BlockHeader, txGroupsAD, int(intra))
+		vBlock, ledgerTxnCount, commitWaitTime, err := g.evaluateBlock(cert.Block.BlockHeader, txGroupsAD, int(intra))
 		if err != nil {
 			return fmt.Errorf("failed to evaluate block: %w", err)
 		}
 		if ledgerTxnCount != g.txnCounter+intra {
 			return fmt.Errorf("evaluateBlock() txn count mismatches theoretical intra: %d != %d", ledgerTxnCount, g.txnCounter+intra)
 		}
 		evaluated = time.Now()
+		g.reportData.Counters[LedgerEvalTimeMS] += uint64(evaluated.Sub(generated).Milliseconds())
```

Comment:
I think there are three numbers to consider:
* the number of transactions including inners
* the number top-level transactions (that the length of the payset)
* the number of atomic transaction groups - which I don't believe we measure anywhere

Reply:
From Conduit's perspective these are all the same thing.

Reply:
I suspect that when we implement batching -for example- we'll want to respect the atomic txn boundary. That is, we could put the database into a "bad state" if we save transactions 1-100 in a block before 101-200, when transactions 98-103 happen to all be part of the same atomic group. So if that's the case, this number is somewhat relevant, though it may not matter so much.

Reply:
`ledgercore.StateDelta` should - hopefully - insulate us from any of those sorts of concerns.